### PR TITLE
Practice app/dockerization 219

### PR DESCRIPTION
**Pull request by mbatuhancelik**
As mentioned in #219, I created a structure with 3 docker containers: mongo, backend, and frontend. All related to the their specific part of the project. 

You can try if this works by running
`docker-compose up --build`

I created .env.production files to create connections between containers, this will be used mostly by backend container to connect to mongo container, Frontend container needs to know the real address of the API. When we deploy the environment, we should change the backend address to a static IP address.

I believe the current version is ready for a review.

### CODE REVIEW

**Review by bahricanyesil — APPROVED** [review]
All other initializations seem ok to me. You successfully created required yml, docker file, .dockerignore and .env.prod files. Good work, thank you!

File: practice-app/frontend/.dockerignore [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Logs
```
[the remark this answers is not in the document]
Comment (mbatuhancelik) [reply practice-app/frontend/.dockerignore:1]: .dockerignore files are identical to .gitignore files in their respective folders, thus, changes between vue and express projects may occur.

They are to make docker work alongside dev builds in our locals and wont have any affect on deployment.

[the remark this answers is not in the document]
Comment (bahricanyesil) [reply practice-app/frontend/.dockerignore:1]: I know that they are specific for back-end and front-end. However, there are some common things like vs-code or apple related files that should be ignored.

[the remark this answers is not in the document]
Comment (mbatuhancelik) [reply practice-app/frontend/.dockerignore:1]: Ah, sure. I will be adding those.

**Pull request merged**